docs/hazmat/backends/interfaces.rst

```diff
@@ -487,3 +487,95 @@ A specific ``backend`` may provide one or more of these interfaces.
 
         :returns: An instance of
             :class:`~cryptography.hazmat.primitives.interfaces.X509Certificate`.
+
+.. class:: DHBackend
+
+    .. versionadded:: 0.8
+
+    A backend with methods for doing Diffie-Hellman key exchange.
+
+    .. method:: generate_dh_parameters(key_size)
+
+        :param int key_size: The bit length of the prime modulus to generate.
+
+        :return: A new instance of a
+            :class:`~cryptography.hazmat.primitives.interfaces.DHParameters`
```

Comment:
Err, it's actually not right now but... *leaves a comment saying we should move those interfaces in the other PR*